custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableau2.json

```diff
@@ -3136,13 +3136,11 @@
       }
     ],
     "sql_settings": {
-      "partition_config": [
-        {
-          "column": "month",
-          "subtype": "date",
-          "constraint": "month"
-        }
-      ]
+      "citus_config": {
```

Comment:
This table is no longer used in the aggregation and will likely be deleted before we roll out citus. Not sure if there is a downside to having it distributed for now, but also not sure it is necessary